pokedex/src/components/Trainers/DelTrainersModal.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+
+export default function DelModal({ show, handleClose, handleDel, national_num }) {
```

Comment:
national_num should be renamed to pk or trainer id, whatever the trainer primary key is